source/grid/grid_out.cc

```diff
@@ -2951,6 +2951,46 @@ namespace
     return v;
   }
 
+  /**
+   * Return all lines of non-internal faces in three dimension.
```

Comment:
I can't see where this is in the file, but the function should be put into an anonymous namespace if it isn't already.

Reply:
It is already!